heron/executor/tests/python/heron_executor_unittest.py

```diff
@@ -157,19 +148,16 @@ def get_expected_instance_command(component_name, instance_id, container_id):
     instance_name = "container_%d_%s_%d" % (container_id, component_name, instance_id)
     return "heron_java_home/bin/java -Xmx320M -Xms320M -Xmn160M -XX:MaxMetaspaceSize=128M " \
            "-XX:MetaspaceSize=128M -XX:ReservedCodeCacheSize=64M -XX:+PrintCommandLineFlags " \
-           "-Djava.net.preferIPv4Stack=true -verbosegc " \
-           "-XX:+UseConcMarkSweepGC -XX:+CMSScavengeBeforeRemark -XX:TargetSurvivorRatio=90 " \
-           "-XX:+PrintGCDetails -XX:+PrintGCTimeStamps -XX:+PrintGCDateStamps " \
-           "-XX:+PrintGCCause -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=5 " \
-           "-XX:GCLogFileSize=100M -XX:+PrintPromotionFailure -XX:+PrintTenuringDistribution " \
-           "-XX:+PrintHeapAtGC -XX:+HeapDumpOnOutOfMemoryError -XX:ParallelGCThreads=4 " \
-           "-Xloggc:log-files/gc.%s.log " \
+           "-Djava.net.preferIPv4Stack=true " \
+           "-XX:+UseG1GC -XX:+ParallelRefProcEnabled -XX:+UseStringDeduplication " \
+           "-XX:MaxGCPauseMillis=100 -XX:InitiatingHeapOccupancyPercent=30 " \
+           "-XX:+HeapDumpOnOutOfMemoryError -XX:ParallelGCThreads=4 " \
```

Comment:
"Xloggc:log-files/gc.%s.log" and the `instance_name` are still necessary I feel since all the logs are in the same folder.

Reply:
`-Xloggc` set of flags are all related to the GC logging. In my testing, I still saw JVM component log files in the `log-files` folder. This change should only remove the GC specific logging (unless someone chooses to enable at submit time).

I believe if the `verbose-gc` flag is not set, the changes I made to the code will not set those JVM flags. This unit test is testing with the defaults, so what you see here matches what was generated by the `heron_executor.py`.

Reply:
Yeah. The flag is for gc logs only, but it separates different gc logs into different files. For example, the gc log for this process would be "log-files/gc.healthmgr.log" and each process has its own gc log file.

Reply:
I think the `heron_executor` functionality that was setting that path and log file names is still there. It's just not there for the default submission. You have to use the flag to enable those parameters. 

These lines of code that you are commenting on are test results for the default case that does not have `verbose_gc` enabled. If there was a test for the `verbose_gc` use case, then we would have another version of this string that included the various GC log parameters. I'm not sure how much more work it will be to add another test case that captures that scenario, but I will look into making it.